### DISCUSSION TITLE
所以这个nb的启动器完全用vb写的？？？

### DISCUSSION THREAD

**Discussion by wantsakboy** [Meloong-Git/PCL#5198] [讨论]
rt

**Comment by Ad-closeNN** [comment]
![image](https://github.com/user-attachments/assets/a6c534cc-48b6-46b2-aed9-aba893a38512)

**Comment by tangge233** [comment]
准确说是 VB.NET，PCL2 开发框架是 .NET Framework 4.6.2，既可以使用 C# 也可以使用 VB.NET 编写，两者编写出来的程序运行效果是差不多的。

**Comment by MacroMeng** [comment]
问一句：PCL需要哪个运行库？有台电脑完全跑不起来（暗指School

**Reply by LinQingYuu** [reply]
[.NET Framework 4.6.2 Runtime](https://dotnet.microsoft.com/zh-cn/download/dotnet-framework/thank-you/net462-offline-installer)

要求操作系统必须为 Windows 7 或更高，建议 x86_64 架构
